core/app/subscribers/spree/order_mailer_subscriber.rb

```diff
@@ -23,13 +19,5 @@ def send_confirmation_email(event)
         order.update_column(:confirmation_delivered, true)
       end
     end
-
-    # Sends reimbursement email to the user
-    #
-    # @param event [Omnes::UnstructuredEvent]
-    def send_reimbursement_email(event)
```

Comment:
This method is not private. What about calling the new method on `ReimbursementMailerSubscriber` and emitting a deprecation warning?

Reply:
Thanks for catching that! Fixed this.